amida-tech/amida-survey-microservice
# Added check for survey status in db to survey patch

#### What's this PR do?
hotfix for an issue where a patch wouldnt go through for a draft survey because only the status included in the patch being sent was considered, not the status of the survey in the database.

#### Related JIRA tickets:
[SER-190](https://jira.amida-tech.com/browse/SER-190)

#### How should this be manually tested?
create a survey patch and make sure it works for a survey that is draft without including a status in the path

#### Any background context you want to provide?
#### Screenshots (if appropriate):
